sql_crud/homework1.txt

```diff
@@ -0,0 +1,15 @@
+#point1
+
+CREATE TABLE new_agents
+AS
+SELECT *
+FROM agents
+WHERE WORKING_AREA = "London"
+
+#point2
+
+CREATE TABLE new_customer
+AS
+SELECT *
+FROM customer
+WHERE CUST_COUNTRY = "UK"
```

Comment:
We don't create table new_customer.
 Insert records into 'new_agents' table from 'agents' table with the following condition:
1. 'WORKING_AREA' of 'agents' table must be 'London'
2. 'CUST_COUNTRY' of 'customer' table must be 'UK'
-> Using LEFT JOIN customer in the above statement